Automattic/wp-calypso
# Nav Redesign: Adjust /sites/ header styles

Related to https://github.com/Automattic/dotcom-forge/issues/6706

## Proposed Changes

This PR pushes the /sites header closer to the Figma.

* Update padding
* Add border-bottom
* Update Header text styles

Before | After
--|--
<img width="1371" alt="Screenshot 2024-04-24 at 5 52 42 PM" src="https://github.com/Automattic/wp-calypso/assets/140841/67079ac9-fc8f-4398-a223-cb1fd19ac57a">  |  <img width="1368" alt="Screenshot 2024-04-24 at 5 52 29 PM" src="https://github.com/Automattic/wp-calypso/assets/140841/d5ad4ff4-4984-431a-b90e-fa56cebd5ddb">

## Testing Instructions




- Apply this PR.
- Go to /sites?flags=layout%2Fdotcom-nav-redesign-v2
- Observe the Sites title is closer to the Figma PZtnZ1nHhnBrsuXDoMJHPD-fi-394%3A97553
- Double check these style updates won't bleed-over somewhere else.


## Pre-merge Checklist
